rtil/src/native/livesplit.rs

```diff
@@ -0,0 +1,49 @@
+use std::sync::Mutex;
+use livesplit_core::{Segment, TimeSpan, Timer, TimingMethod};
+use once_cell::sync::Lazy;
+
+static LIVESPLIT_STATE: Lazy<Mutex<LiveSplit>> = Lazy::new(|| Mutex::new(LiveSplit::new()));
```

Comment:
I don't think this should be in its own global. Instead, I think it'd fit better into the `rebo::STATE`.